src/components/utils.js

```diff
@@ -0,0 +1,10 @@
+export function getSearchItemInfo() {
+  const { area_type, city, state_code } = state.currentSearchItem;
+  return `${area_type}, ${city}, ${state_code}`;
+}
```

Comment:
Pass state as param, don't use it globally
function getSearchItemInfo(state) {
  const { area_type, city, state_code } = state.currentSearchItem;
  return `${area_type}, ${city}, ${state_code}`;
}